automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -23,7 +24,7 @@ class MethodConfigSpec extends FreeSpec with Matchers with LazyLogging
   val noExpressionErrorText: String = "Error: Method configuration expects an entity of type sample, but you gave us an entity of type sample_set."
   val missingInputsErrorText: String = "is missing definitions for these inputs:"
 
-  "launch a simple workflow" in withWebDriver { implicit driver =>
+  "launch a simple workflow" taggedAs Tags.SmokeTest in withWebDriver { implicit driver =>
```

Comment:
this test method is identical to """launch a method config from the method repo""". instead removing, tagged this as a smoketest.